zstd-safe/zstd-sys/build.rs

```diff
@@ -200,7 +200,9 @@ fn main() {
     }
 
     // println!("cargo:rustc-link-lib=zstd");
-    let (defs, headerpaths) = if cfg!(feature = "pkg-config") {
+    let (defs, headerpaths) = if cfg!(feature = "pkg-config")
+        && !cfg!(feature = "static")
```

Comment:
I think we should make it an error to have both pkg-config and static features, rather than silently dropping one.

Reply:
I prefer `static` to override `pkg-config`, since it is possible for one dependency to request `pkg-config` and another to request `static`.

In this situation, it is not always possible to modify or it maybe hard to remove that and I would prefer statically linked to dynamically linked zstd.

Reply:
If a dependency asks for pkg-config, we should honor this decision and not discard it.

Though in general asking for `pkg-config` is not something re-usable libraries should be doing, it's more of a end-of-the-line binary decision.